### ISSUE TITLE
python 3.6 connect python 2.7 TypeError: 'b'instancemethod'' object is not callable

### ISSUE THREAD

**Issue by getixin** [tomerfiliba/rpyc#250]
##### Environment

- rpyc version
- python version
- operating system

##### Minimal example

Server:
- python version 2.7.13
python rpyc_classic.py

Client:
- python version 3.6.2
```python
import rpyc
c = rpyc.classic.connect("xxx")
```

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python36\lib\site-packages\rpyc\utils\classic.py", line 68, in connect
    return factory.connect(host, port, SlaveService, ipv6 = ipv6, keepalive = keepalive)
  File "C:\Python36\lib\site-packages\rpyc\utils\factory.py", line 91, in connect
    return connect_stream(s, service, config)
  File "C:\Python36\lib\site-packages\rpyc\utils\factory.py", line 52, in connect_stream
    return connect_channel(Channel(stream), service = service, config = config)
  File "C:\Python36\lib\site-packages\rpyc\utils\factory.py", line 41, in connect_channel
    return Connection(service, channel, config = config)
  File "C:\Python36\lib\site-packages\rpyc\core\protocol.py", line 159, in __init__
    self._init_service()
  File "C:\Python36\lib\site-packages\rpyc\core\protocol.py", line 164, in _init_service
    self._local_root.on_connect()
  File "C:\Python36\lib\site-packages\rpyc\core\service.py", line 147, in on_connect
    self._conn.builtin = self._conn.modules.builtins
  File "C:\Python36\lib\site-packages\rpyc\core\service.py", line 118, in __getattr__
    return self[name]
  File "C:\Python36\lib\site-packages\rpyc\core\service.py", line 115, in __getitem__
    self.__cache[name] = self.__getmodule(name)
TypeError: 'b'instancemethod'' object is not callable

The python version must be the same?

**Comment by coldfix** [comment]
Hi, yes connection between py2 and py3 is not possible. There is also some explanation here:

https://rpyc.readthedocs.io/en/latest/install.html#cross-interpreter-compatibility

Best, Thomas

**Comment by ixFelix** [comment]
devices: 

- windows 10 with python 2.7.13 
- EV3 Brick with unknown python version
- both with rpyc version 3.3.0

I followed this tutorial:
[https://ev3dev-lang.readthedocs.io/projects/python-ev3dev/en/stable/rpyc.html#](url)

Hello, I get the same error message. I installed python 2.7.13 on my PC (windows 10), because that is the python version, ev3 uses, when I type "python" in the putty terminal. As I understand it, the ev3 brick still uses python 3 and I don't know how to make the it use python 2.7.13. In the tutorial it says, that in rpyc_server.sh I should type python3
